src/utils/rest-api.ts

```diff
@@ -537,6 +540,27 @@ export function createContingencyList(
         body: JSON.stringify(formContent),
     });
 }
+export function createFilterBasedContingency(
+    name: string,
+    description: string,
+    filtersUuids: FilterAttributes[],
```

Comment:
```suggestion
    contingency: FilterBasedContingencyList,
```
Should define in contingency-list.type.ts type FilterBasedContingencyList taken from the backend dto (actions-server)